test/module-test.cc

```diff
@@ -63,3 +73,507 @@ TEST(module_test, macros) {
+  EXPECT_EQ("42", fmt::to_string(short{42}));
+  EXPECT_EQ("42", fmt::to_string(unsigned short{42u}));
+  EXPECT_EQ("42", fmt::to_string(42L));
+  EXPECT_EQ("42", fmt::to_string(42uL));
+  EXPECT_EQ("42", fmt::to_string(42LL));
+  EXPECT_EQ("42", fmt::to_string(42uLL));
+  EXPECT_EQ("42", fmt::to_string(42.0));
+  EXPECT_EQ("42", fmt::to_string(42.0f));
+  EXPECT_EQ("42", fmt::to_string(42.0L));
+  EXPECT_EQ("42", fmt::to_string("42"));
+  EXPECT_EQ("a", fmt::to_string('a'));
+  EXPECT_EQ("97", fmt::to_string(unsigned char{'a'}));
+  EXPECT_EQ("97", fmt::to_string(signed char{'a'}));
+  EXPECT_EQ("true", fmt::to_string(true));
+  EXPECT_EQ("0x2a", fmt::to_string((void *)42));
+}
+
+TEST(module_test, format) {
+  EXPECT_EQ("42", fmt::format("{}", 42));
+  EXPECT_EQ("42", fmt::format("{:}", 42u));
+  EXPECT_EQ("+42", fmt::format("{:+d}", short{42}));
+  EXPECT_EQ("42", fmt::format("{:d}", unsigned short{42u}));
+  EXPECT_EQ("-42", fmt::format("{0}", -42L));
+  EXPECT_EQ("2a", fmt::format("{0:x}", 42uL));
+  EXPECT_EQ("***42", fmt::format("{:*>5}", 42LL));
+  EXPECT_EQ("0X2A", fmt::format("{:#X}", 42uLL));
+  EXPECT_EQ("===42.000000===", fmt::format("{:=^15f}", 42.0));
+  EXPECT_EQ("00000042.1", fmt::format("{:<010.5G}", 42.1f));
+  EXPECT_EQ(" +4.21235e+11", fmt::format("{:+{}.{}e}", 42.1234567e10L, 13, 5));
+  EXPECT_EQ("-0x1.5000000000000p+5", fmt::format("{: a}", -42.0));
+  constexpr auto nan = std::numeric_limits<double>::quiet_NaN();
+  EXPECT_EQ("-nan-", fmt::format("{:-^5}", nan));
+  constexpr auto inf = std::numeric_limits<double>::infinity();
+  EXPECT_EQ("-INF-", fmt::format("{:-^5F}", inf));
+  EXPECT_EQ("-INF ", fmt::format("{: 5G}", -inf));
+  EXPECT_EQ("42", fmt::format("{:s}", "42"));
+  EXPECT_EQ("a", fmt::format("{:c}", 'a'));
+  EXPECT_EQ("a", fmt::format("{:c}", unsigned char{'a'}));
+  EXPECT_EQ("a", fmt::format("{:c}", signed char{'a'}));
+  EXPECT_EQ("false", fmt::format("{}", false));
+  EXPECT_EQ("0B1", fmt::format("{:#B}", true));
+  EXPECT_EQ("0x2a", fmt::format("{:p}", (void *)42));
+  EXPECT_EQ("1234", fmt::format("{:L}", 1234));
+}
+
+TEST(module_test, format_to) {
+  std::string s;
+  fmt::format_to(std::back_inserter(s), "{}", 42);
+  EXPECT_EQ("42", s);
+
+  char buffer[4] = {0};
+  fmt::format_to(buffer, "{}", 42);
+  EXPECT_EQ("42", std::string_view(buffer));
+}
+
+TEST(module_test, formatted_size) {
+  EXPECT_EQ(2u, fmt::formatted_size("{}", 42));
+}
+
+TEST(module_test, format_to_n) {
+  {
+    std::string s;
+    auto result = fmt::format_to_n(std::back_inserter(s), 1, "{}", 42);
+    EXPECT_EQ(2u, result.size);
+    EXPECT_EQ("4", s);
+  }
+  {
+    char buffer[4] = {0};
+    auto result = fmt::format_to_n(buffer, 3, "{}", 12345);
+    EXPECT_EQ(5u, result.size);
+    EXPECT_EQ(buffer + 3, result.out);
+    EXPECT_EQ("123", std::string_view(buffer));
+  }
+}
+
+template <typename... Args>
+auto create_args(Args&&... args) {
+  return fmt::make_format_args<fmt::format_context>(args...);
+}
```

Comment:
I suggest using `make_format_args` directly instead of introducing a wrapper function.